src/bun.js/bindings/bindings.zig

```diff
@@ -4324,6 +4324,17 @@ pub const JSValue = enum(JSValueReprInt) {
         });
     }
 
+    pub fn values(_: JSValue, globalThis: *JSGlobalObject, value: JSValue) JSValue {
+        return cppFn("values", .{
+            globalThis,
+            value,
+        });
+    }
+
+    pub fn getKeys(_: JSValue, globalThis: *JSGlobalObject, value: JSC.JSValue) JSValue {
```

Comment:
instead of `getKeys`, I would use `keys()` and change the signature to look like: `pub fn keys(value: JSValue, globalThis: *JSGlobalObject) JSValue`